bokeh/command/subcommands/serve.py

```diff
@@ -388,10 +459,10 @@
     )),
 
     ('--use-config', dict(
-        metavar='CONFIG',
-        type=str,
-        help="Use a YAML config file for settings",
-        default=None,
+        metavar = 'CONFIG',
+        type    = str,
+        help    = "Use a YAML config file for settings",
+        default = None,
```

Comment:
Judging by this PR, the style for keyword arguments' values alignment is different throughout Bokeh codebase. And I've never seen such alignment in any Python project before, although that might be because of my limited exposure to other Python projects. Is there any prettifier config or documentation that makes it clear that this style is preferred?

Reply:
No presently, ideally we can just switch to black maybe at some point and forget about debates. I would never write kwargs in an actual function call this way, but although these are ultimately used that way, here they are more like data literals and that is the case where this is my personal pref (see also: lining up dict literal values)

Reply:
In this case I mostly just wanted to make things consistent even within the same file